### PR TITLE
Added venv installation steps for Windows

### PR DESCRIPTION
**Pull request by ludeeus**
**Description:**
Adds python virtual environment installation instruction for Windows

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by dshokouhi** [comment]
Looks good to me, better direction for users to go with a virtual environment install.

File: source/_docs/installation/windows.markdown [hunk 1]
```diff
@@ -15,6 +15,8 @@ To run Home Assistant on Microsoft Windows installation you need to install Pyth
 There may be alpha or beta releases of Python listed on that download page (marked by the letters `a` or `b` in the version number. Do not use these versions.
 </p>
 
+If you use you machine for something other than Home Assistant you should install it in a [python virtual environment](#Installing_in_a_python_virtual_environment)
+
```
Comment (fabaff) [inline source/_docs/installation/windows.markdown]: Please fix the typo and add a period to the end of the sentence. It's preferred to use a capital `P` for Python.

File: source/_docs/installation/windows.markdown [hunk 2]
```diff
@@ -39,3 +41,23 @@ C:\...\> pip3 install --upgrade homeassistant
 ```
 
 Then start Home Assistant.
+
+## {% linkable_title Installing in a python virtual environment %}
+Open CMD.exe and paste the folowing commands (one at the time):
```
Comment (fabaff) [inline source/_docs/installation/windows.markdown]: Please fix the typo. As a non-windows user I would guess that it's `cmd.exe`.

File: source/_docs/installation/windows.markdown [hunk 3]
```diff
@@ -39,3 +41,23 @@ C:\...\> pip3 install --upgrade homeassistant
...
+C:\...\> py -m venv homeassistant
+C:\...\> cd homeassistant
+C:\...\> Scripts\activate.bat
+C:\...\> py -m pip install wheel setuptools pip
+C:\...\> py -m pip install homeassistant
+C:\...\> hass --open-ui
+```
+
+## {% linkable_title Upgrading in a python virtual environment %}
+To update Home Assistant, stop it and then open CMD.exe and paste the folowing commands (one at the time):
```
Comment (fabaff) [inline source/_docs/installation/windows.markdown]: Dito.

File: source/_docs/installation/windows.markdown [hunk 4]
```diff
@@ -39,3 +41,23 @@ C:\...\> pip3 install --upgrade homeassistant
...
+
+## {% linkable_title Installing in a python virtual environment %}
+Open CMD.exe and paste the folowing commands (one at the time):
+```bash
+C:\...\> py -m venv homeassistant
+C:\...\> cd homeassistant
+C:\...\> Scripts\activate.bat
+C:\...\> py -m pip install wheel setuptools pip
+C:\...\> py -m pip install homeassistant
+C:\...\> hass --open-ui
```
Comment (fabaff) [inline source/_docs/installation/windows.markdown]: `--open-ui` is only useful if you want to browse the Home Assistant frontend the local machine. Should be removed here.

**Review by fabaff — CHANGES_REQUESTED** [review]

**Review by fabaff — DISMISSED** [review]
:bird:

**Review by fabaff — APPROVED** [review]

**Pull request merged**